src/Sylius/Bundle/PromotionBundle/Form/DataTransformer/PercentFloatToLocalizedStringTransformer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Sylius\Bundle\PromotionBundle\Form\DataTransformer;
+
+
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\Form\Extension\Core\DataTransformer\PercentToLocalizedStringTransformer;
+
+class PercentFloatToLocalizedStringTransformer extends PercentToLocalizedStringTransformer
+{
+    /**
+     * Transforms between a percentage value into a float
+     *
+     * @param string $value Percentage value
+     *
+     * @return float Normalized value
+     *
+     * @throws TransformationFailedException if the given value is not a string or
+     *                                       if the value could not be transformed
+     */
```

Comment:
I think this docblock is redundant, the name of the class speaks for itself.

Reply:
I think it is good to keep it for at least the part with `@throws` in order to know which exception can be thrown by this method